src/features/activities/reactions/ReactionOptions.tsx

```diff
@@ -91,9 +91,6 @@ export const ReactionOptions = ({
             <ReactionButton
               color="reaction"
               key={r}
-              onClick={() =>
```

Comment:
I think its best practice to leave the onClick on the top level button. Otherwise, it's possible you click on the button but the input isn't clicked and so the UI doesn't behave as expected.

So even though this commit fixes it, I think we should understand why and where the other addReaction is being triggered from, and fix this in another way, rather than just moving the onClick to the input. @sshmm